operations/validate_csm_health.md

```diff
@@ -61,335 +54,100 @@ Health Check scripts can be run:
+```
+Review the output for `Result: FAIL` and follow the instructions provided to resolve any such test failures. With the exception of the [Known Test Issues](#autogoss-issues), all health checks are expected to pass.
+
+The ncn-healthcheck suite covers the following (as appropriate for worker, master and storage nodes):
+* Commands Available
+* Interfaces Has IP
+* Bond Members Have Links
+* Interfaces Has VLAN
+* Bond Members Have Correct MTU
+* Nodes DNS Resolvable
+* Unbound Nameserver In resolv.conf
+* Sealed Secrets Key Exists
+* Kubernetes StorageClasses
+* Kubernetes Velero Backup Storage Locations Ready
+* Kubernetes Velero Daily Vault Backup Schedule Exists
+* Kubernetes Velero No Failed Backups
+* Spire Health Check
+* Kubernetes Query BSS Cloud-init for spire meta data
+* Spire Pods
+* Verify spire-agent is enabled and running
+* Validate token key ID exists in spire jwks
+* Verify spire api fetch of jwt
+* Service 'etcd' Running
+* Etcd ca.crt file exists
+* Etcd client crt file exists
+* Etcd client key file exists
+* Validate the Kubernetes etcd on this node is healthy
+* Default Gateway Same as CAN Gateway
+* Verify cray etcd is healthy
+* Validate that the local platform CA bundle matches the one in cloud-init
+* Check clock skew on k8s and storage nodes
+* Validate PodSecurityPolicy is enabled
+* Resolve external DNS
+* Kubernetes Check cray-vault Vault Cluster Status
+* Kea has Active DHCP Leases
+* All Kubernetes Etcd Clusters have a Backup Created within the Last 24 Hours
+* All Etcd Pods in a Cluster are Running on a Different Worker Node
+* Etcd Database Health
+* Etcd Cluster Endpoint Health
+* Validate the kubernetes etcd cluster has three active members
+* Service 'etcd' Running
+* Validate the Kubernetes etcd on this node is healthy
+* Switch BGP Neighbors
+
+The ncn-kubernetes-checks suite covers the following (as appropriate for worker and master nodes):
+* Weave IPAM Health
+* Kubernetes Pod IPs in NMN Pool
+* Kubernetes ConfigMaps
+* Kubernetes Pods
+* LoadBalancer Services Have External IP
+* Kubernetes Cert-manager Certificates Ready
+* Kubernetes Cert-manager Issuers Ready
+* Kubernetes Ensure Configmaps Containing CA Certs Exist
+* Kubernetes Check cray-vault Vault Cluster Status
+* Unbound Hostname Exists
+* Worker Node CONRUN FS Label
+* Worker Node CONLIB FS Label
+* Worker Node K8SLET FS Label
+* Service 'etcd' Running
+* Etcd ca.crt file exists
+* Etcd client crt file exists
+* Etcd client key file exists
+* Validates the dedicated drive for ETCD is set up correctly on master nodes (ETCDLVM FS Label)
+* Validate the Kubernetes etcd on this node is healthy
 
-Known failures and how to recover:
+<a name="autogoss-issues"></a>
+#### 1.1.1 Known Test Issues
 
+* K8S Test: Kubernetes Query BSS Cloud-init for ca-certs
+  - May fail immediately after platform install. Should pass after the TrustedCerts Operator has updated BSS (Global cloud-init meta) with CA certificates.
```

Comment:
We should use regular English for these sentences.

Reply:
done